browser/src/control/jsdialog/Definitions.Types.ts

```diff
@@ -165,6 +165,8 @@ interface ToolItemWidgetJSON extends WidgetJSON {
 	noLabel?: boolean;
 	command: string; // command to trigger options for a panel
 	text: string; // title to show or for tooltip
+	icon: string; // url to an svg
```

Comment:
```suggestion
	icon?: string; // url to an svg
```
If I am not mistaken this is optional.

Reply:
> If I am not mistaken this is optional.
At least your initial example didn't need it to apply an icon:

yeah, technically it's optional (the button can be created without it) but without the "icon" parameter the button will look like this:

<img width="877" height="233" alt="screenshot_05082025_180459" src="https://github.com/user-attachments/assets/0f1b5f8b-74e8-447f-b464-b8b0c947078b" />


In my initial example, the "unoCommand" was an actual uno command which had an svg file of the same name, that's why the icon was set.